web/src/app/urql.ts

```diff
@@ -104,15 +105,19 @@ const apolloRefetchExchange: Exchange = ({ forward }) => {
   }
 }
 
-export const client = createClient({
-  url: pathPrefix + '/api/graphql',
-  suspense: true,
-  exchanges: [
-    refetchExchange(),
-    cacheExchange,
-    apolloRefetchExchange,
-    retryExchange({}) as Exchange,
-    fetchExchange,
-  ],
-  requestPolicy: 'cache-and-network',
-})
+export function newClient(url: string): Client {
+  return createClient({
```

Comment:
is there any benefit from breaking this variable up? seems to just add lines from the previous definition

Reply:
not sure I know what you mean, for the new decorator we need to be able to create a new client, but in all other cases we need a single instance.

storybook uses `newClient` and the normal goalert webapp uses the const

Reply:
The code is exactly the same- I didn't notice the `export` being the only difference. Makes sense. 👍